src/main/io/displayport_max7456.c

```diff
@@ -82,6 +83,18 @@ static int drawScreen(displayPort_t *displayPort) {
     return 0;
 }
 
+void setBackgroundBlack() {
+    if (displayPortProfileMax7456()->useBlackBackgroundInMenus) {
+        max7456BackgroundBlack();;
+    }
+}
+
+void setBackgroundTransparent() {
+    if (displayPortProfileMax7456()->useBlackBackgroundInMenus) {
```

Comment:
Im not a huge fan of putting the check to see if the screen should turn black here. Then in cms.c it will look like you aste always turning the osd black. Either rename the function so we know that it doesn't always set it black or do this check in cms.c